python/services/github_service.py

```diff
@@ -477,21 +476,22 @@ def fetch_all_repositories_in_org(self) -> list[dict[str, Any]]:
         after_cursor = None
         repos = []
 
+        #  Disable logging. The output is too verbose and not required.
+        logging.getLogger("gql").setLevel(logging.CRITICAL)
+        logging.disable(logging.INFO)
```

Comment:
Is it possible to centralise this configuration?

Reply:
We do in this [file](https://github.com/ministryofjustice/operations-engineering/blob/main/python/config/logging_config.py)

Reply:
To clarify, is there anything for me to do here? I suppose I could pass the log level to the class?

Reply:
I think overriding the logging level in the code is okay, I decreased the log level of gql in github service in my PR yesterday as well to Warning

Reply:
just realised your change is in github service as well: my [commit](https://github.com/ministryofjustice/operations-engineering/commit/809604f4a18f2c18355f5aa8187d3472b094af2b#diff-5823a7548d7e01e3aae750d3bdba9709e4f3746567c18d72e51bf3eef7f09e3eL20) puts it at the top level